plugins/infobox/src/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import uuid from 'uuid'
+
+// You are obviously not limited to material-ui, but we really enjoy
+// the material-ui icons!
+import InfoIcon from 'material-ui-icons/Info'
+
+const InfoboxPlugin = ({ children }) => (
+    <div className="ory-editor-plugins-infobox">
+        { children }
+    </div>
+)
+
+export default ({ defaultPlugin }) => ({
+    Component: InfoboxPlugin,
+    IconComponent: <InfoIcon />,
+    name: 'example/layout/infobox',
```

Comment:
You could change this to something like `schul-cloud/layout/infobox` or `serlo/layout/infobox` if you like. Not that important atm, but might be more descriptive.